esmvaltool/diag_scripts/ocean/diagnostic_tools.py

```diff
@@ -0,0 +1,176 @@
+"""Python example diagnostic."""
+import logging
+import os
+import sys
+
+import iris
+import matplotlib.pyplot as plt
+
+# This part sends debug statements to stdout
+logger = logging.getLogger(os.path.basename(__file__))
+logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
+
+
+def folder(name):
+    """
+        This snippet takes a string, makes the folder and the string.
+        It also accepts lists of strings.
+        """
+    if isinstance(name, list):
+        name = '/'.join(name)
+    if name[-1] != '/':
+        name = name + '/'
+    if os.path.exists(name) is False:
+        os.makedirs(name)
+        logger.info('Making new directory:' + str(name))
+    return name
+
+
+def get_input_files(cfg, index=0):
+    """Get a dictionary with input files from metadata.yml files."""
+    metadata_file = cfg['input_files'][index]
+    with open(metadata_file) as file:
+        metadata = yaml.safe_load(file)
+    return metadata
+
+
+def sensibleUnits(cube, name):
+    """
```

Comment:
Instead of forcing the units, it will be nice to have them as a configuration in the diagnostic